pkgs/applications/misc/blender/default.nix

```
@@ -71,7 +72,7 @@
   pkg-config,
   potrace,
   pugixml,
-  python311Packages, # must use instead of python3.pkgs, see https://github.com/NixOS/nixpkgs/issues/211340
+  python312Packages, # must use instead of python3.pkgs, see https://github.com/NixOS/nixpkgs/issues/211340
```

```suggestion
  python3Packages, # must use instead of python3.pkgs, see https://github.com/NixOS/nixpkgs/issues/211340
```
python3Packages is python312Packages

also version overrides should be done in all-packages.nix

https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md#changing-implicit-attribute-defaults

---

Please see the comment right next to it

---

the comment about "python3.pkgs" ? my comment is about `python3Packages`

---

ah, gotcha. Yes, good point.